site/content/en/latest/tasks/security/backend-tls.md

```diff
@@ -159,7 +182,7 @@ Note: SectionName is an optional field that specifies the name of the port in th
 If the target is a [Backend] resource, the `sectionName` field should be set to the port number of the backend.
 
 {{< tabpane text=true >}}
-{{% tab header="Apply from stdin" %}}
+{{% tab header="ConfigMap" %}}
```

Comment:
curious why this changed

Reply:
we need to support both ConfigMap/ClusterTrustBundle.

TBH, that `apply from file` seems useless.